packages/op-datasets/src/op_datasets/etl/intermediate/models/daily_address_summary.py

```diff
@@ -1,27 +1,34 @@
+# -*- coding: utf-8 -*-
 import duckdb
+from op_coreutils.duckdb_inmem import init_client
 
 from op_datasets.etl.intermediate.registry import register_model
 from op_datasets.etl.intermediate.task import IntermediateModelsTask
-from op_datasets.etl.intermediate.udfs import Expression, wei_to_eth, wei_to_gwei, safe_div, to_sql
+from op_datasets.etl.intermediate.udfs import (
+    Expression,
+    wei_to_eth,
```

Comment:
it is embarassing how happy these make me.

alternatively / additionally. I wonder if we can just pull things from [web3.py](https://web3py.readthedocs.io/en/stable/)? It has similar conversion functions & I think also has abi decoding for the future